extensions/wikia/IndexingPipeline/MySQLMetricWorkerConnectionBase.class.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace Wikia\IndexingPipeline;
+
+class MySQLMetricWorkerConnectionBase extends ConnectionBase {
+
+	public function __construct() {
+		global $MySQLMetricWorker;
+		$this->host = $MySQLMetricWorker[ 'host' ];
```

Comment:
Please rename it wgMySQLMetricWorker to conform to the convention. This needs to find its place in config repo too :)